docs/development/development.md

```diff
@@ -80,68 +80,38 @@ A `*` denotes a required variable.
 | `$TEMPDIR`        | `/tmp/tmp.RANDOM`                  |   This is the randomly generated working dir for BitOps.  |
 | `$ROOT_DIR`        | `/opt/bitops_deployment`                  |   This working dir for BitOps (moved to `$TEMPDIR` during execution)  |
 
-> For more information about the core variables, see
-> [scripts/deploy.sh](https://github.com/bitovi/bitops/blob/master/scripts/deploy.sh)
+### Standard BitOps Packages
 
-### Standard BitOps Packages and Libraries:
-
-System Packages natively available in a running BitOps container:
```

Comment:
A note here is that we should mention the usable binaries only, which are really important for the user and skip everything else.

References to the libraries and pip packages in requirements.txt can quickly become outdated and it's just a noise for the user.